Assets/Treasure/TDK/Runtime/Analytics/TDK.Analytics.cs

```diff
@@ -32,11 +32,25 @@ public void OnApplicationPause_Analytics(bool isPaused)
         internal void SetTreasureConnectInfo(string smartWalletAddress, int chainId)
         {
 #if !DISABLE_TREASURE_ANALTYICS
-            TDKServiceLocator.GetService<TDKAnalyticsService>().SetTreasureConnectInfo(smartWalletAddress, chainId);
+            try
+            {
+                TDKServiceLocator.GetService<TDKAnalyticsService>().SetTreasureConnectInfo(smartWalletAddress, chainId);
+            }
+            catch (System.Exception ex)
+            {
+                TDKLogger.LogException("Error setting connect info for TDKAnalyticsService", ex);
+            }
 #endif
 
 #if TDK_HELIKA
-            TDKServiceLocator.GetService<TDKHelikaService>().SetTreasureConnectInfo(smartWalletAddress, chainId);
+            try
+            {
+                TDKServiceLocator.GetService<TDKHelikaService>().SetTreasureConnectInfo(smartWalletAddress, chainId);
+            }
+            catch (System.Exception ex)
+            {
+                TDKLogger.LogException("Error setting connect info for TDKHelikaService", ex);
+            }
```

Comment:
Errors from helika being misconfigured were showing up in the connect screen and preventing login, so i wrapped all calls here to make sure they dont throw errors.

Also, added the new LogException helper, while in the editor it will double log, one for the message and one for the original exception so the stack trace can be seen.